rest/accounts/v1/credentials_aws.go

```diff
@@ -152,6 +154,43 @@ func (c *ApiService) ListCredentialAws(params *ListCredentialAwsParams) (*ListCr
 	return ps, err
 }
 
+//Retrieve a single page of CredentialAws records from the API. Request is executed immediately.
+func (c *ApiService) CredentialAwsPage(params *ListCredentialAwsParams, pageToken string, pageNumber string, pageSize string) *client.Page {
+	path := "/v1/Credentials/AWS"
+
+	data := url.Values{}
+	headers := make(map[string]interface{})
+
+	if params != nil && params.PageSize != nil {
+		data.Set("PageSize", fmt.Sprint(*params.PageSize))
+	}
+
+	data.Set("PageToken", pageToken)
+	data.Set("PageNumber", pageNumber)
+	data.Set("PageSize", pageSize)
```

Comment:
pageSize coming from 2 places?

Reply:
hmmm do you think it's a good idea to add the `PageToken` and `PageNumber` as a list parameter as well since it already has a `PageSize` parameter?

Reply:
No, if you're listing or streaming then you don't control which page you're on. Just the size of each page and limiting the total items.